### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by shelden1992**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,6 +7,18 @@
     <groupId>groupId</groupId>
     <artifactId>educationEpam</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>7</source>
```
Comment (valeriiafatova) [inline pom.xml]: why 7?

File: src/main/java/part1/ConverterBinaryToDecimal.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+import java.util.Scanner;
+
+public class ConverterBinaryToDecimal {
+    public static void main(String[] args) {
+
+        try (Scanner scanner = new Scanner(System.in)) {
+            System.out.println("Enter binary number");
+            String binaryString = null;
+            while (!(binaryString = scanner.nextLine()).equalsIgnoreCase("exit") && !binaryString.isEmpty()) {
```
Comment (valeriiafatova) [inline src/main/java/part1/ConverterBinaryToDecimal.java]: extract condition to separate method

File: src/main/java/part1/ConverterBinaryToDecimal.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package part1;
```
Comment (valeriiafatova) [inline src/main/java/part1/ConverterBinaryToDecimal.java]: Format all files in all tasks with ctrl + alt + L

File: src/main/java/part1/ConverterBinaryToDecimal.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                        step++;
+                    }
+                    System.out.printf("Your's binary number %s is %d in decimal \n", binaryString, decimal);
+                    System.out.println("************************************ \n");
+                    System.out.println("If you whant escape enter \"exit\" or press enter or continue convert");
+                } catch (IllegalArgumentException e) {
+                    System.out.println("Please try again, you enter wrong binary number");
+                }
+            }
+
```
Comment (valeriiafatova) [inline src/main/java/part1/ConverterBinaryToDecimal.java]: extra lines? remove

File: src/main/java/part1/ConverterBinaryToDecimal.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+        }
+
+    }
+
+
+}
+/**
+ * or System.out.println(Integer.parseInt(binaryString, 2)), but it's not interesting
```
Comment (valeriiafatova) [inline src/main/java/part1/ConverterBinaryToDecimal.java]: :) it is forbidden

File: src/main/java/part5/СhangeBitInPosition.java [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        System.out.println(changeBitForZeroInPosition(number, position));
+
+
+    }
+
+    private static int changeBitForOneInPosition(int number, int position) {
+        try {
+            String s = Integer.toBinaryString(number);
+            if ((position + 1 > s.length() || position < 0)) {
+                throw new IllegalArgumentException();
```
Comment (valeriiafatova) [inline src/main/java/part5/СhangeBitInPosition.java]: See no sence throw exception and in 4 lines catch it. Log it and return 0 directly instead

File: pom.xml (hunk 1 above)
Comment (shelden1992) [reply pom.xml]: default value, change to 1.8

**Pull request merged**